### DISCUSSION TITLE
Unable to configure OpenCL with Imagemagick 6.9

### DISCUSSION THREAD

**Discussion by Pupa-git** [ImageMagick/ImageMagick6#143] [General]
Hello all. I have a problem when assembling Imagemagick 6.9.12-4 using OpenCL on a Nvidia Tesla V100

I attach the output of commands before and after make install
Perhaps you can tell me what the problem?
![Снимок экрана от 2021-03-23 20-39-39](https://user-images.githubusercontent.com/74722221/112487275-a6023600-8d8d-11eb-86f3-0d01e0597041.png)
![Снимок экрана от 2021-03-23 20-40-05](https://user-images.githubusercontent.com/74722221/112487290-a8fd2680-8d8d-11eb-9f8f-0c6a857a1042.png)
![Снимок экрана от 2021-03-23 20-49-10](https://user-images.githubusercontent.com/74722221/112487295-aa2e5380-8d8d-11eb-9b7f-b014da0b0403.png)

**Comment by urban-warrior** [comment]
Is your host big-endian?  OpenCL support requires little-endian byte order on your host.
